src/App.jsx

```diff
@@ -1,19 +1,22 @@
 import "./App.css";
 import { Routes, Route } from "react-router-dom";
+import { useState } from "react";
 import Header from "./components/Header";
 import NavBar from "./components/NavBar";
 import ReviewsList from "./components/ReviewsList";
 import ReviewInfo from "./components/ReviewInfo";
 import LogIn from "./components/LogIn";
 
 function App() {
+  const [reviews, setReviews] = useState([])
```

Comment:
This state it held too high - it should be in ReviewsList

Reply:
on my plan i have it here because i intend to share it with the nav bar so i can show specific reviews based on their category